tests/integration_tests/api/auth_methods/test_kubernetes.py

```diff
@@ -208,9 +208,9 @@ def test_read_role(self, label, create_role_first=True, raises=None, exception_m
             )
 
     @parameterized.expand([
-        # param(
-        #     'success',  # TODO: figure out why this is returning a "InvalidPath" exception ("unsupported path")
-        # ),
+        param(
+            'success',  # TODO: figure out why this is returning a "InvalidPath" exception ("unsupported path")
```

Comment:
if this test is functional now, can you remove the comment please.

`# TODO: figure out why this is returning a "InvalidPath" exception ("unsupported path")`

Reply:
done!